pack/tfa/hote_encounter.json

```diff
@@ -1,24 +1,25 @@
 [
     {
-        "back_text": "[skull]: -2 (-4 instead if you are in a [[cave]] location).<br/>[cultist]: -3. If you fail, place 1 doom on your location.<br/>[tablet]: -3. If you are poisoned, this test automatically fails instead. If you are not poisoned and you fail, put a set-aside Poisoned weakness into play in your threat area.<br/>[elder_thing]: -4. If you fail, take 1 horror.",
+        "back_text": "<b>Hard / Expert</b>\n[skull]: -2 (-4 instead if you are in a [[Cave]] location).\n[cultist]: -3. If you fail, place 1 doom on your location.\n[tablet]: -3. If you are poisoned, this test automatically fails instead. If you are not poisoned and you fail, put a set-aside Poisoned weakness into play in your threat area.\n[elder_thing]: -4. If you fail, take 1 horror.",
         "code": "04205",
         "double_sided": true,
         "encounter_code": "heart_of_the_elders",
+        "encounter_position": 1,
         "faction_code": "mythos",
         "illustrator": "—————",
         "name": "Heart of the Elders",
         "pack_code": "hote",
         "position": 205,
         "quantity": 1,
-        "text": "[skull]: -1 (-3 instead if you are in a [[cave]] location).<br/>[cultist]: -2. If you fail, place 1 doom on your location.<br/>[tablet]: -2. If you are poisoned, this test automatically fails instead.<br/>[elder_thing]: -3. If you fail, take 1 horror.",
+        "text": "[skull]: -1 (-3 instead if you are in a [[Cave]] location).\n[cultist]: -2. If you fail, place 1 doom on your location.\n[tablet]: -2. If you are poisoned, this test automatically fails instead.\n[elder_thing]: -3. If you fail, take 1 horror.",
         "type_code": "scenario"
     },
     {
-        "back_text": "[action]: <b>Resign.</b> \"Let's make camp and solve this puzzle tomorrow.\"<br/>[action]: Check your supplies. If you have a compass, look at the top 3 cards of the exploration deck. Place 1 on the bottom and the other 2 on top, in any order.",
```

Comment:
I think this should just be 'text' rather than 'back_text' - @Kamalisk ?

Reply:
I moved it down to `text` and made this a`linked`, I think the diff looks a bit weird.

Basically both sides of the location are 'revealed' and have different number of clues, so I think we need to use linking here?